types/block_meta.go

```diff
@@ -25,6 +27,47 @@ func NewBlockMeta(block *Block, blockParts *PartSet) *BlockMeta {
 	}
 }
 
+func (bm *BlockMeta) ToProto() *tmproto.BlockMeta {
+	if bm == nil {
+		return nil
+	}
+
+	pb := &tmproto.BlockMeta{
+		BlockID:   bm.BlockID.ToProto(),
+		BlockSize: int64(bm.BlockSize),
```

Comment:
why do we need to cast here again? because bm.BlockSize is int?

Reply:
yup yup, need to go through and size up the ints